README.md

```diff
@@ -258,6 +258,31 @@ kube-system namespace:
 checkov -d . --skip-check kube-system
 ```
 
+Run a scan of a container image:
+```sh
+checkov --framework sca_image --docker-image sha256:1234example --dockerfile-path /Users/path/to/Dockerfile --bc-api-key ... --repo-id test/test_docker
```

Comment:
the repo-id is non mandatory if I recall correctly

Reply:
Good point. It's not. Removed.